incubator/telegraf/telegraf/values.yaml

```diff
@@ -0,0 +1,77 @@
+# SLATE Telegraf Monitoring Configuration
+
+
+# The slate instance value.
+Instance: ""
+
+
+# The number of pods to spin up.
+replicaCount: 1
+
+
+# The image the application will use.
+# Leave this as-is.
+image:
```

Comment:
The catalog generally does not allow the image to be set in values. It should be hard coded in the template. See the comment from above.